Update/add Diamond to intro chapter (#310)
AvolitesLtd/TitanManual: docs/about-the-consoles/diamond.md

---
id: diamond
title: Diamond 9

> sidebar_label: Diamond

Reviewer: ```suggestion
sidebar_label: Diamond 9
```

description: Diamond - console design and features
---

![Diamond9](/docs/images/Diamond-Main.png)

The Diamond 9 has the following main control areas:

> \(A\) The three main **Touch screens** contain fixture, palette and group select

Author: Yes, I was just trying to give some examples of the sort of controls you might see there. The same language is used on the other consoles in this section.

buttons. They also show legends for the playback faders and the
current wheel functions at the bottom of the right hand screen.

\(B\) The **Playback faders** and **Rotary playback faders**

>  select and control cues, programs and other configurable functions.

Author: should probably say cues, cue lists and chases

\(C\) The **System Prompt and Menu Softkeys** have their own screen.

\(D\) The **Programming and setup controls** configure and program the console,
within which the **attribute touchscreen** shows and sets fixture attributes and 

> the **macro/executor buttons** allow storage of frequent key

Author: Yes, again all the other consoles say this too, I think from the early days of Titan when they really were just key macro buttons. I will update this and the others.

sequences, screen layouts, groups or programs. To the left of these
controls are the cue list playback controls.

\(E\) Three **Preview screens** are provided for monitoring
video feeds or can be used as extra touch displays.

## Touch screen controls

![Diamond-F-G](/docs/images/Diamond-F.png)

\(F\) The **Workspace windows** on the touch screens contain touch buttons for
fixtures, palettes, groups, playbacks, macros and more in a user-configurable
layout. You can also set fixture attributes and show information windows.

\(G\) The **System Prompt and Menu Softkeys** screen is the nerve centre of
the console and shows you what is going on. This display contains touch
buttons for the main menu which change depending on what the console
is doing. Softkey commands are shown in the manual 
like this: \[Edit Times\]

## Playbacks

![Diamond-H](/docs/images/Diamond-H.png)

\(H\) The **Rotary Faders** at the top can be used to store
a range of program functions including cues, chases, and fixture groups.
The rotary faders have a level indicator around them which is shown in

> a user-configurable colour (halo colour).

Author: I was thinking the same, links would be good for all the different consoles, but I will add as a separate PR as it is quite a lot of work.

The touch screen below the faders shows legends indicating their function.

\(I\) **The Page Select buttons** in the
centre of the faders let you change the left or right fader banks to a different page.

\(J\) Two more sets of **Rotary Faders** are provided above the Playback Faders.

\(K\) The motorised **Playback faders** are the main place to store and play back cues or chases
but can also be assigned a range of other functions. Each fader has a level indicator
which can be set to user-configurable halo colour.
The touch screen above the faders shows information about

> each playback.

Author: I am not sure whether to put this here. It is a nice feature but does it need explaining in the manual?

> each playback.

Reviewer 2: Fair enough!

\(L\) The **Page Select buttons** let you change the left or right
fader banks to a different page
of playbacks (for both the playback faders and the lower rotary faders). 

\(M\) The **Macro Buttons** store frequently used sequences of keypresses
which can be replayed with a single button press. Playbacks can also be
stored here. The display immediately above the buttons shows legends for
the buttons. There is a smaller bank of 6 buttons between the playback faders with
their own legend screen.

\(N\) The **Playback controls** are used to control the playback of cue

> lists and include a configurable T-bar for the Scene Master function and motorised A-B faders.

Reviewer 2: ```suggestion
lists and include a T-bar to control the Scene Master and a motorised fader pair
in the Sequence Control Section to control the connected playback/master.
```

Reviewer 2: Maybe also worth mentioning the scene master status screen

Author: Would you not call these A-B faders, that is how I would know them on other consoles, I suppose from the old 2-preset days.

Reviewer 2: Mm... at release, they don't support separate In/Out A/B crossfading. The left fader is playback level and the right fader is manual crossfade (whole cue) or effect rate multiplier... the plan is for it to eventually support this but yeah it doesn't really support AB-type functionality as yet, sadly...

Author: Ah OK, I will use your description.

## Programming controls

![Diamond-N](/docs/images/Diamond-O.png)

> \(O\) The five  **Control wheels** are used to set control values (attributes) for

Reviewer 2: Might be worth mentioning the new wheel layer paging buttons to the right of the wheels which allow you to select a wheel layer from Levels / Fade+Delay / Shape parameters / Visualiser / Connected Playback.

Author: I think this would be better explained in a later section as this intro is really for the novice to see what is where, and those concepts are yet to be explained. I will put it in when I update the wheels section.

Reviewer 2: I suppose my thinking was that theyr'e a whole new bank of buttons and they're not covered by any of the red boxes - maybe adding a new bullet point for them?

Author: OK how about "The buttons to the right of the wheels switch them to other functions such as fade timing,
shape control or visualiser control."

Reviewer 2: sgtm

the fixtures, and to set chase speeds and fades. The touch screen above
the wheels shows information about the attributes being controlled.

\(P\) The dedicated **Attribute Touchscreen** shows the Attribute Editor
window giving a different way to select attributes such as colours and gobos.

\(Q\) The **Function buttons** are used to carry out functions such as storing
cues, copying, patching, saving to disk, etc.

\(R\) The **Attribute Buttons** are used to select which attributes of
a fixture (e.g. colour, gobo, pan, focus) are going to be controlled
using the Control wheels. The buttons light up to show you which
attributes are active.

\(S\) The **Numeric keypad** and other control buttons are used to enter
values and change controls on the console.

\(T\) The dedicated **Intensity Wheel** and display controls intensity
of selected fixtures.

\(U\) The **trackball** controls pan and tilt of fixtures, and can also be
used to control a mouse pointer for the screens.

## The Back Panel

![Diamond Back Panel](/docs/images/Diamond-Back.png)

\(a\) Mains power input (PowerCON TRUE1), mains isolator and UPS interrupt switch. Do not

> power the console off using this mains isolator switch.

Reviewer 2: I don't think we've told users where the power switch is, though? :P

Reviewer: It is mentioned in the section called "Front edge of the console"

Author: It's described in the section just above this "Front edge of the console" which I added as a later commit, you might not have got it.

\(b\) Additional USB sockets for general purpose use.

> \(c\) DMX input for data capture.

Reviewer: You can't really capture data with the dmx input so probably best not to word it like this. I think it's basically just for triggers.

\(d\) Outputs for two additional screens - HDMI and USB for touch screen.

\(e\) MIDI in, thru and out.

> \(f\) Digital S/PDIF connection.

Reviewer 2: ```suggestion
\(f\) Digital S/PDIF audio output.
```
